docs/install.rst

```diff
@@ -14,8 +14,9 @@ Installation
 Introduction
 ============
 
-This part of the documentation covers the installation of CrateDB on Linux,
-macOS and Windows systems.
+This part of the documentation covers the installation of CrateDB on Linux and
+macOS.
```

Comment:
Is it worth stating explicitly that Windows is not supported?

```suggestion
This part of the documentation covers the installation of CrateDB on Linux and
macOS. Windows installations are not supported.
```

Reply:
from what I gather, it's more a case of if someone wants to use Windows, we'd prefer it's something they bring up with us directly so we can advise them on a case-by-case basis. I might be wrong! but my reasoning was that an explicit "we don't support windows" is perhaps not strictly the case (i.e., more that we'd prefer not to advertise installation on windows)

but happy for @seut to correct me or provide more info!